lib/openfoodfacts.dart

```diff
@@ -428,14 +432,44 @@ class OpenFoodAPIClient {
         path: 'api/v1/predict/ingredients/spellcheck',
         queryParameters: spellingCorrectionParam);
 
-    Response response = await HttpHelper()
-        .doGetRequest(spellingCorrectionUri, user: user, queryType: QueryType.PROD);
+    Response response = await HttpHelper().doGetRequest(spellingCorrectionUri,
+        user: user, queryType: QueryType.PROD);
     SpellingCorrection result = SpellingCorrection.fromJson(
         json.decode(utf8.decode(response.bodyBytes)));
 
     return result;
   }
 
+  /// Extract the ingredients from image with the given parameters.
+  /// The ingredients language should be given (ingredients_fr, ingredients_de, ingredients_en)
+  /// Returns the ingredients using OCR.
+  /// By default the query will use the Google Cloud Vision.
+  /// By default the query will hit the PROD DB
+
+  static Future<OcrIngredientsResult> extractIngredients(
+      User user, String barcode, OcrIngredientsField ocrIngredientsField,
```

Comment:
Instead of a OcrIngredientsField, we can just ask for the 2 letter language code of the ingredients. Or for the corresponding OpenFoodFactsLanguage. That way we can completely remove the class for OcrIngredientsField. Less code.